### PR TITLE
aireplay-ng: add Hindi translation

### PR DESCRIPTION
**Pull request by Avtech04**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.hi/common/aireplay-ng.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# aireplay-ng
+
+> वायरलेस नेटवर्क में पैकेट इंजेक्ट करें। एयरक्रैक-एनजी का हिस्सा।
```
Comment (Magrid0) [inline pages.hi/common/aireplay-ng.md:3]:
```suggestion
> वायरलेस नेटवर्क में पैकेट इंजेक्ट करें। एयरक्रैक-एनजी का हिस्सा।.
```

**Review by Magrid0 — APPROVED** [review]
Looks good to me, thanks for your contribution!

File: pages.hi/common/aireplay-ng.md (hunk 1 above)
Comment (kbdharun) [inline pages.hi/common/aireplay-ng.md:3]:
```suggestion
> वायरलेस नेटवर्क में पैकेट इंजेक्ट करें। एयरक्रैक-एनजी का हिस्सा।
> `aircrack-ng` का हिस्सा।
```

File: pages.hi/common/aireplay-ng.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+> अधिक जानकारी: <https://www.aircrack-ng.org/doku.php?id=aireplay-ng>.
+
+- एक्सेस प्वाइंट के मैक पते, क्लाइंट के मैक पते और एक इंटरफ़ेस को देखते हुए एक विशिष्ट संख्या में असंबद्ध पैकेट भेजें:
```
Comment (kbdharun) [inline pages.hi/common/aireplay-ng.md:6]:
```suggestion
- एक्सेस प्वाइंट के MAC पते, क्लाइंट के MAC पते और एक इंटरफ़ेस को देखते हुए एक विशिष्ट संख्या में असंबद्ध पैकेट भेजें:
```
I would suggest leaving abbreviations/capitalized terms like this untranslated.

**Review by kbdharun — COMMENTED** [review]
Welcome to tldr and thanks for your contribution. I have some suggestions for this page.

File: pages.hi/common/aireplay-ng.md (hunk 2 above)
Comment (Avtech04) [reply pages.hi/common/aireplay-ng.md]: ok

File: pages.hi/common/aireplay-ng.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# aireplay-ng
+
+> वायरलेस नेटवर्क में पैकेट इंजेक्ट करें। `aireplay-ng` का हिस्सा।
```
Comment (kbdharun) [inline pages.hi/common/aireplay-ng.md:3]:
```suggestion
> वायरलेस नेटवर्क में पैकेट इंजेक्ट करें।
> `aireplay-ng` का हिस्सा।
```

**Comment by Avtech04** [comment]
anything else?

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Pull request merged**